### ISSUE TITLE
Version 6 stopped working outside LAN

### ISSUE THREAD

**Issue by leochiessa** [kilmajster/ngrok-spring-boot-starter#69]
Hello @kilmajster, I was using your ngrok-spring-boot-starter version 6, until yesterday when it stopped working outside of my LAN. Any ideas? Thank you so much.

**Comment by kilmajster** [comment]
Hi @leochiessa, is there any reason that you stick with old version? did you tried with latest `0.9.0` ?

**Comment by leochiessa** [comment]
No, there isn't. But if I use v0.9.0, ngrok doesn't even run... could it be
'cause i'm using spring v2.2.2?

**Comment by kilmajster** [comment]
yup, that's because versions/APIs incompatibly between spring version used in latest version of this project and yours. Later on I'll update README with some kind of starter<->spring versions map, but if you are using spring `2.2.2.RELEASE` then you should be good with v`0.7.0` of this starter.

**Comment by leochiessa** [comment]
Thanks for answering, I changed to version 0.7.0 and had no effects...
